src/CookiePolicyBanner/index.jsx

```diff
@@ -20,18 +20,14 @@ class CookieBanner extends Component {
   }
 
   componentDidMount() {
-    if (hasViewedCookieBanner()) {
-      this.toggleDisplay(false);
-    } else {
-      createHasViewedCookieBanner();
-      this.toggleDisplay(true);
-    }
+    this.toggleDisplay(!hasViewedCookieBanner());
   }
 
   onClose(event) {
-    this.setState({ open: false });
-
-    this.props.onClose(event);
+    this.setState({ open: false }, () => {
```

Comment:
I want to explicitly call out this callback - [`setState` takes a callback](https://reactjs.org/docs/react-component.html#setstate) to guarantee that the update has been applied, since `setState` is async and may not immediately update the component.